frappe/utils/background_jobs.py

```diff
@@ -37,7 +37,7 @@ def enqueue(method, queue='default', timeout=None, event=None,
 	'''
 	# To handle older implementations
 	if 'async' in kwargs:
```

Comment:
Maybe just replace the entire `if` condition with:

```suggestion
	is_async = kwargs.pop("async", True)
```

Reply:
Everyday we learn something new :+1: